draft-packages/select/KaizenDraft/Select/Select.tsx

```diff
@@ -174,6 +176,10 @@ const DropdownIndicator: typeof components.DropdownIndicator = props => (
   </components.DropdownIndicator>
 )
 
+const LoadingMessage: typeof components.LoadingMessage = props => (
+  <components.LoadingMessage {...props} className={styles.loadingMessage} />
+)
+
```

Comment:
I've added this in so we can control  the styles of the loading message wrapper. It will still render the default "Loading..." but just give us the ability to set the default styles to inter.